examples/Cassie/osc/high_level_command.h

```diff
@@ -8,8 +8,8 @@
 #include "drake/systems/framework/leaf_system.h"
 
 namespace dairlib {
-namespace cassie {
-namespace osc {
+    namespace cassie {
```

Comment:
Same issue with the indentation here. The indentation also makes it difficult to see the actual changes.